src/axe-raw-sarif-converter.ts

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { isEmpty } from 'lodash';
```

Comment:
>lodash [](start = 25, length = 6)

We're using lodash here in product code, so it should be listed as a dependency in package.json.

Reply:
What file would you put that function in?